client/src/js/components/bhDepotDropdown.js

```diff
@@ -22,9 +22,10 @@ function bhDepotController(Depot, AppCache, Notify) {
     .then(function (rows) {
       if (!rows.length) { return; }
       $ctrl.depots = rows;
-      $ctrl.selection = cache.selection || $ctrl.depots[0];
+      $ctrl.depotsAvailable = !!rows.length;
```

Comment:
This is really nit-picky, but it is helpful to name boolean variables with `is` or `has` as the prefix to show they are boolean.  In this case, it would be nice to call this `$ctrl.hasDepotsAvailable`.

The reason to do this is because `is` or `has` expects a true/false response.  `isOkay?` can only be a "yes" or "no".  Similarly, `hasDepotsAvailable?` is either a "yes" or "no".  However, `depotsAvailable` could easily be an array of depots which are available to use.

Just best practice :)